packages/gatsby-cli/src/index.js

```diff
@@ -3,22 +3,24 @@
 // babel-preset-env doesn't find this import if you
 // use require() with backtick strings so use the es6 syntax
 import "@babel/polyfill"
+const semver = require(`semver`)
 
 const createCli = require(`./create-cli`)
 const report = require(`./reporter`)
 
-const version = process.version
-const verDigit = Number(version.match(/\d+/)[0])
-
 const pkg = require(`../package.json`)
 const updateNotifier = require(`update-notifier`)
 // Check if update is available
 updateNotifier({ pkg }).notify()
 
-if (verDigit < 6) {
+const MINIMUM_NODE_VERSION = `8.0.0`
```

Comment:
Would be to nice to couple this with the version of Gatsby (core) or something. Not sure of the best approach for that--shouldn't block this PR, but something to think about.

Reply:
Maybe engines in gatsby/package.json?

Reply:
(I went with the engines approach for now!)